modules/food-menu/food-item-detail.js

```diff
@@ -0,0 +1,118 @@
+// @flow
+
+import * as React from 'react'
+import {Text, ScrollView, View, StyleSheet} from 'react-native'
+import {Column, Row} from '@frogpond/layout'
+import {ListRow, ListSeparator, Detail, Title} from '@frogpond/lists'
+import * as c from '@frogpond/colors'
+import map from 'lodash/map'
+import {DietaryTags} from './dietary-tags'
+import {type TopLevelViewPropsType} from '../../source/views/types'
+import type {
+	MenuItemType as MenuItem,
+	NutritionDetailType,
+	MasterCorIconMapType,
+} from './types'
+
+type Props = TopLevelViewPropsType & {
+	navigation: {state: {params: {icons: MasterCorIconMapType, item: MenuItem}}},
+}
+
+export class MenuItemDetailView extends React.Component<Props> {
+	static navigationOptions = {
+		title: 'Nutrition',
+	}
+
+	render() {
+		const item = this.props.navigation.state.params.item
+		const icons = this.props.navigation.state.params.icons
+
+		const calculateAmount = (nutrition: NutritionDetailType) => {
+			// turn "lessthang" and "lessthanmg" into "less than 1g", "less than 5mg", etc
+			return nutrition.unit.includes('lessthan')
+				? `less than ${nutrition.value}${nutrition.unit.split('lessthan')[1]}`
+				: `${nutrition.value}${nutrition.unit}`
+		}
+
+		return (
+			<ScrollView style={styles.container}>
+				<Text selectable={true} style={styles.title}>
+					{item.label}
+				</Text>
+
+				<DietaryTags
+					corIcons={icons}
+					dietary={item.cor_icon}
+					isDetail={true}
+					style={styles.iconContainer}
+				/>
+
+				{item.description ? (
+					<View>
```

Comment:
Use `<React.Fragment>` instead of `<View>` if all you need to do is wrap two elements